test/signs_ui/config/sign_group_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule SignsUi.Config.SignGroupTest do
+  use ExUnit.Case
+  alias SignsUi.Config.SignGroup
+
+  describe "expired/3" do
+    test "does not expire a sign group without an alert or expiration time" do
+      assert not SignGroup.expired?(
+               %SignGroup{},
+               DateTime.new!(~D[2021-05-21], ~T[17:31:00]),
+               MapSet.new(["active_alert"])
+             )
+    end
+
+    test "does not expire a sign group with an active alert id" do
```

Comment:
You can also use `refute` instead of `assert not`. https://hexdocs.pm/ex_unit/1.12/ExUnit.Assertions.html#refute/1

Reply:
Oh nice, thanks!